Teleportation/Tests.qs

```diff
@@ -189,6 +189,28 @@ namespace Quantum.Kata.Teleportation {
         TeleportPreparedStateTestLoop(PrepareAndSendMessage_Reference, ReconstructAndMeasureMessage);
     }
     
+    @Test("QuantumSimulator")
+    operation T18_EntanglementSwapping () : Unit {
+        
+        let numRepetitions = 100;
+
+        for i in 0 .. numRepetitions {
+            use (qAlice1, qAlice2) = (Qubit(), Qubit());
+            H(qAlice1);
```

Comment:
You can use Entangle_Reference here and below to make the code shorter and more self-documenting

Reply:
done